app/client-v2/src/stores/projectStore.ts

```diff
@@ -0,0 +1,132 @@
+import { getApiUrl } from "@/config";
+import {
+  COMPLETE_STATUS_TYPES,
+  type AnalysisStatus,
+  type ApiResponse,
+  type Project,
+  type ProjectSample,
+  type WorkerResponse,
+  WorkerResponseValueTypes
+} from "@/types/projectTypes";
+import { mande } from "mande";
+import { defineStore } from "pinia";
+import { Md5 } from "ts-md5";
+
+const baseApi = mande(getApiUrl(), { credentials: "include" });
+
+// TODO: add proper error handling. Maybe best to add error state attribute and watch accordingly cos of nested things interval/workers
+export const useProjectStore = defineStore("project", {
+  state: () => ({
+    basicInfo: {} as Pick<Project, "id" | "name" | "timestamp">,
+    fileSamples: [] as ProjectSample[],
+    projectHash: "",
+    isRun: false,
```

Comment:
Does this mean has started running, or has completed running?  Maybe `started` would be better if the former.